lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -280,12 +304,21 @@ def _configure_nodes(self, environment: Environment, log: Logger) -> None:
             node_context.cloud_init_file_path = os.path.join(
                 vm_disks_dir, f"{node_context.vm_name}-cloud-init.iso"
             )
-            node_context.os_disk_base_file_path = qemu_node_runbook.qcow2
+
+            node_context.os_disk_base_file_path = os.path.join(
+                vm_disks_dir, os.path.basename(qemu_node_runbook.qcow2)
+            )
+            self.host_node.shell.copy(
```

Comment:
The OS image can potentially be multiple gigabytes in size. So, we should avoid copying it in the local host case, to improve performance.